### ISSUE TITLE
Documentation doesn't mention that you have to call setAuth

### ISSUE THREAD

**Issue by pkrumins** [nodejitsu/node-cloudservers#13]
$subject 

Took me a while to figure it out!

I was following the docs and the docs seem to show that all you have to do is create a client via cloudservers.createClient. Turns out that doesn't do the auth but just store the auth info in this.config.

**Comment by jfhbrook** [comment]
Hey Peteris! Long time no see. Sup dawg?

Okay, so in order to actually auth you have to call `.setAuth` even though you pass the actual auth info in with the call to `createClient`? Interesting.

Assuming this isn't some mistake, we'd accept any and all edits to the readme, if you feel up to it.

**Comment by pkrumins** [comment]
Hey Josh!

Doing good. Hacking on Browserling together with James!

Oh actually I made a mistake and it does authenticate automatically. I had a bug before (this one: https://github.com/nodejitsu/node-cloudservers/issues/12#issuecomment-5184417). And I changed two places at once - I added setAuth and I added that host. It made me think it didn't automatically authenticate.

Sorry about that. I'm closing this issue.
